docker-compose.yml

```diff
@@ -7,10 +7,10 @@
 # Should be used together with `docker/docker-compose.prod.yml`
 # in production.
 
-version: "3.6"
+version: "3.6.1"
 services:
   db:
-    image: "postgres:9.6.9-alpine"
+    image: "postgres:15.2-alpine"
```

Comment:
Just a couple of things I noted in relation to the postgres upgrade:
1. The databases don't seem to be compatible between the two, so I had to delete mine to get it to work. Does there need to be a migration strategy? For me it wasn't a problem because there was nothing in the database I was concerned about keeping, so I just deleted it.
2. The latest postgres docker image doesn't seem to allow `POSTGRES_PASSWORD` to be left blank. Would it be worth adding a note to the `config/.env.template` file about this?

Reply:
1. Indeed, one would need to update the storage that the Docker container creates to the newer postgres version. This is possible and would be necessary if we had production data, but I guess as we don't at this stage we're good! 😉 
2. Indeed, good point to leave a comment!